onnx_caffe2/backend.py

```diff
@@ -179,18 +179,21 @@ def tensor2list(onnx_tensor):
             # Use the onnx.numpy_helper because the data may be raw
             return onnx.numpy_helper.to_array(onnx_tensor).flatten().tolist()
 
-        if onnx_tensor.data_type == TensorProto.FLOAT:
+        if onnx_tensor.data_type in [TensorProto.FLOAT16,
+                                     TensorProto.FLOAT,
+                                     TensorProto.DOUBLE]:
```

Comment:
This looks wrong. I looked at Caffe2's tensor proto and they have a doubles field: https://github.com/caffe2/caffe2/blob/master/caffe2/proto/caffe2.proto#L46 which I think we should probably use here. Otherwise we are not accurately importing them

Reply:
Yep, either GivenDoubleTensorFill, or load TensorProtos as blobs, both would work. :-)